lib/Graphs/PAG.cpp

```diff
@@ -70,6 +113,7 @@ const std::string GepValPN::toString() const {
     std::string str;
     raw_string_ostream rawstr(str);
     rawstr << "GepValPN ID: " << getId() << " with offset_" + llvm::utostr(getOffset());
+    rawstr << "\n";
```

Comment:
I mean all the newlines should be removed for `toString` methods since we need to debug in the terminal. Two lines make it a bit tricky.

Reply:
Hi Richard, maybe could you please share the figures before and after removing the `\n`, so that we can know which way would be better? Thanks,